src/components/RatingAndReviews/ReviewsList.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import { StatusEnum } from '../../hooks/ReviewStars/useReviewsRating';
+import ReviewCard from './ReviewCard.jsx'; // eslint-disable-line
+
+const Wrapper = styled.div`
+  flex: 2;
+  min-height: 200px; // FIXME:
+  max-height: 100vh;
+  overflow: scroll;
+`;
+
+const UnorderedList = styled.ul`
+  border: 1px solid;
+  list-style-type: none;
+`;
+
+function ReviewsList({ productId }) { // eslint-disable-line
+
+  const [reviews, setReviews] = useState([]);
+  const [page, setPage] = useState(1);
+  const [sort, setSort] = useState('relevant');
+  const [haveMoreReviews, setHaveMoreReviews] = useState(true);
+  const [status, setStatus] = useState(StatusEnum.idel);
+  const [error, setError] = useState(null);
+
+  const reviewsFetchController = new AbortController();
+  useEffect(() => {
+    if (!productId) return () => {};
+    setStatus(StatusEnum.pending);
+    axios.get('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/reviews', {
```

Comment:
The reason I didn't use useServerFetch here was because I would like to pass params directly to axios, or the eslint warn me the url line was too long. useServerFetch do not support pass params directly now (And I don't think that is necessary.), so I use axios directly for now.

Reply:
If you want to refactor to use useServerFetch, there is always the option to pass in params using a template literal as the endpoint. This works fine too though.

Reply:
Same way you did in line 32 of useReviewsRating